test/models/WritableReadableEnumTests.scala

```diff
@@ -37,7 +37,7 @@ trait WritableReadableEnumTests {
     }
   }
 
-  def writableTest[A](enum: A, expectedResult: String, testName: String)(implicit formatter: Format[A]): Unit = {
+  def writableTest[A](`enum`: A, expectedResult: String, testName: String)(implicit formatter: Format[A]): Unit = {
     testName in {
       val result = Json.toJson(enum)
```

Comment:
Not sure how this works post-change, however best to change this to:
```suggestion
      val result = Json.toJson(`enum`)
```